### ISSUE TITLE
We need to add support for the CUE file that contains multiple files

### ISSUE THREAD

**Issue by Gateswong** [Gateswong/GatesMusicPet#1]
Some CUE file contains files like: 

```
FILE "file1.flac"
    TRACK 01 AUDIO
        TITLE "Go ahead"
        INDEX 01 00:00:00
FILE "file2.flac"
    TRACK 02 AUDIO
        TITLE "No problem!"
        INDEX 01 00:00:00
```

`Meta` class shall be able to handle the multiple files.

If necessary, the converter can do conversion based on the field of `"original_file"` in the `Meta` instance.

**Comment by Gateswong** [comment]
.

**Comment by Gateswong** [comment]
Resolved by pull #5 .
